query-engine/query-engine/src/features.rs

```diff
@@ -0,0 +1,53 @@
+use crate::opt::PrismaOpt;
+
+/// Feature models the engine feature toggles that can be set up during the startup via options
+/// See [PrismaOpt]'s boolean flags, for each of the corresponding feature toggles.
+#[enumflags2::bitflags]
+#[derive(Clone, Copy, Debug)]
+#[repr(u8)]
+pub enum Feature {
```

Comment:
Could this be `pub(crate)` instead?

Reply:
`EnableFeatures` type is a public alias, I'm affraid I can't hide this while keeping the `EnableFeatures` public :(